ports/nrf/boards/ohs2020_badge/mpconfigboard.mk

```diff
@@ -0,0 +1,10 @@
+USB_VID = 0x239A
+USB_PID = 0x8072
+USB_PRODUCT = "OHS2020 Badge"
+USB_MANUFACTURER = "Adafruit Industries LLC"
```

Comment:
This should be you/your company.

Reply:
Well it is a community design so there was no company but maybe I can just use my company.

Reply:
Community name is fine!

Reply:
OSHWA it is. Resolved with the latest commit.